test/python/algorithms/eigensolvers/test_vqd.py

```diff
@@ -200,6 +200,24 @@ def store_intermediate_result(eval_count, parameters, mean, metadata, step):
         np.testing.assert_array_almost_equal(history["mean"], ref_mean, decimal=2)
         np.testing.assert_array_almost_equal(history["step"], ref_step, decimal=0)
 
+    @data(H2_PAULI, H2_OP, H2_SPARSE_PAULI)
+    def test_optimal_values(self, op):
+        """Test running same VQD twice to re-use optimizer, then switch optimizer"""
```

Comment:
The description here seems to be a copy/paste from the following test, but...

...I guess the check for this field could have been added as a subtest under the test_basic_operator above that seems to have subtests for a bunch of the result fields, rather than a new test which has to run the VQD computation again.

Reply:
yes, you're totally right, I overlooked that. Fixed in [bfa1bac](https://github.com/Qiskit/qiskit-terra/pull/10279/commits/bfa1bacccfc2ba7de5f0ee2fb7029b92a2c1ec1d)